test/unit/test_vector.c

```diff
@@ -198,6 +198,28 @@ static bool test_vector_clone(void) {
 	mu_end;
 }
 
+static bool test_vector_sort(void) {
+	RzVector *v = rz_vector_new(sizeof("aaa"), NULL, NULL);
+	rz_vector_push(v, "caa");
+	rz_vector_push(v, "abb");
+	rz_vector_push(v, "ccc");
```

Comment:
please also add another element which is the same as an existing one. Just to see how the algo behaves.